constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintLayout.kt

```diff
@@ -186,6 +236,35 @@ internal fun rememberConstraintLayoutMeasurePolicy(
     }
 }
 
+@Composable
+@PublishedApi
+internal fun rememberMotionLayoutMeasurePolicy(
+    optimizationLevel: Int,
+    constraintSetStart: ConstraintSet,
+    constraintSetEnd: ConstraintSet,
+    progress: Float,
+    measurer: MotionMeasurer
+) = remember(optimizationLevel, constraintSetStart, constraintSetEnd, progress) {
```

Comment:
I think it's better to remove progress from the remember here, as this way the MeasurePolicy will be recreated on every frame.
I would do it like this on lines 196:
```
val progressState = remember { mutableStateOf(0f) }
SideEffect { progressState.value = progress }
```
Then pass this state to `rememberMotionLayoutMeasurePolicy` and use `progressState.value` as parameters to `performInterpolationMeasure` and `performLayout`

Reply:
You might also need to move measurer.clear somewhere else.

Reply:
good idea, done.